Feature/dx 98 create glossary list (#125)
SwedbankPay/developer.swedbankpay.com: resources/terminology.md

| **Access Token**                | The OAauth 2 Access token needed to access Swedbank Pay eCom APIs. Tokens are generated in Swedbank Pay Admin. Learn how to getting started in the Admin Manual.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |

> | **Account Receivable Consumer** | The [`AccountReceivableConsumer`][invoice-url] API is the fundament for Swedbank Pay [Invoice Payments][invoice-url] service. It is a service where Swedbank Pay helps produce and distribute invoices to consumers/end-users.                                                                                                                                                                                                                                                                                                                                                                                                                                              |

Reviewer: It's not obvious from this description how `AccountReceivableConsumer` is different to `FinancingConsumer`. Both of these concepts should be more thorougly explained in Invoice Payments Other Features. Can you add a task for it?

Author: Yes, creating a task for 1) and 2) and invoice payments other features 3)

---
title: Swedbank Pay Resources
sidebar:
  navigation:
  - title: Resources
    items:
    - url: /resources/
      title: Introduction
    - url: /resources/test-data
      title: Test Data
    - url: /resources/demoshop
      title: Demoshop
    - url: /resources/development-guidelines
      title: Open Source Development Guidelines
    - url: /resources/terminology
      title: Terminology
---

{% include alert-development-section.md %}

{:.table .table-striped}
| Term  | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |

> | :------ | :------------ |

Reviewer 2: Auto format table `Alt + Shit + F`

| **Access Token**                                 | The OAauth 2 Access token needed to access Swedbank Pay eCom APIs. Tokens are generated in [Swedbank Pay Admin][admin-portal]. Learn how to getting started in the [Admin Manual][admin-manual].                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |

> | **Account Receivable Consumer**                 | The [`AccountReceivableConsumer`][invoice-url] API is the fundament for Swedbank Pay [Invoice Payments][invoice-url] service. It is a service where Swedbank Pay helps produce and distribute invoices to consumers/end-users.                                                                                                                                                                                                                                               |

Reviewer 2: It's a bit odd to have two identical links in the same sentence.

| **Payee**                     | The company that receive funds for the payment.                                                                                                                                                                                                                                                                                                                                                                     |

> | **Payee ID**                 | The ID of the company that receive funds for the payment. May also be known as Merchant ID.                                                                                                                                                                                                                                                                                                                                                                                                                     |

Reviewer 2: ```suggestion
| **Payee ID**                 | The ID of the company that receive funds for the payment. Also referred to as Merchant ID.                                                                                                                                                                                                                                                                                                                                                                                                                     |
```

| **Verify**     | The payment operation that initiates a [verification][verify-url] payment process. It is a payment that lets you post verifications to confirm the validity of card information, without reserving or charging any amount. This option is used to generate a payment- or recurrence token, that can be used in a recurring payments scenarios or for one-clickpayments, without charging the card in the process. |

[admin-manual]: https://developer.payex.com/xwiki/wiki/developer/view/Main/ecommerce/resources/admin/
[admin-portal]: https://admin.externalintegration.payex.com/psp/login

> [admin-getting-started]: https://developer.payex.com/xwiki/wiki/developer/view/Main/ecommerce/resources/admin/

Reviewer 2: We don't need to link to info about the "old" admin in the old developer portal pages.

[admin-interface-search]: https://developer.payex.com/xwiki/wiki/developer/view/Main/ecommerce/resources/admin-account-interface-and-search/
[auto-capture]: /payments/card/other-features#capture
[hateoas]: https://en.wikipedia.org/wiki/HATEOAS
[invoice-url]: /payments/invoice/
[two-phase-payments]: /payments/#the-fundamentals
[callback-url]:/payments/card/other-features#callback

> [cancel-url]: /checkout/after-payment#cancel

Reviewer 2: Why is this the only link that goes to `/checkout` when the other links to specific pages go to `/payments/card`?

Author: changed

> [checkout-url]: payments/checkout/

Reviewer 2: This link does not work.

Author: fixed

> [common-headers]: payments/#headers

Reviewer 2: This link does not work.

Author: fixed

[restful-api]: payments/#connection-and-protocol
[verify-url]: /payments/card/other-features#verify
[one-phase-url]: /payments/#the-fundamentals
[payout]: /payments/card/other-features#payout
[recur]: /payments/card/other-features#recur

> [sale-authorization-url]: /payments/#creating-an-authorization-or-sales-transaction

Reviewer 2: This link does not work.

Author: fixed
